libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -3286,18 +3286,18 @@ string YulUtilFunctions::copyStructToStorageFunction(StructType const& _from, St
 		_to.identifier();
 
 	return m_functionCollector.createFunction(functionName, [&]() {
+		YulUtilFunction function;
+		function.name = functionName;
+		function.arguments = {"slot", "value"};
```

Comment:
You could do

```c++
return YulUtilFunction{functionName, {"slot", "value"}, {}, templ.render()};
```

The downside however would be that the parameters would end up away from function body so I'm not sure if saving a few lines is really worth it...

Reply:
For this particular case, I think it is better to leave as is because of reason you explain.
But in general (in many cases) we can do:
```
return YulUtilFunction{
	functionName, 
	{"slot", "value"}, 
	{}, 
	Whiskers(
	...
	).render()
};
```